tslearn/neighbors.py

```diff
@@ -125,8 +128,8 @@ class KNeighborsTimeSeries(KNeighborsTimeSeriesMixin, NearestNeighbors):
     metric_params : dict or None (default: None)
         Dictionnary of metric parameters.
         For metrics that accept parallelization of the cross-distance matrix
-        computations, `n_jobs` key passed in `metric_params` is overridden by
-        the `n_jobs` argument.
+        computations, `n_jobs` and `verbose` keys passed in `metric_params` 
+        are overridden by the `n_jobs` and `verbose` arguments.
```

Comment:
verbose has to be added to that class (doc + __init__ method) (cf TravisCI failure).